cli/cmd/cli_unix.go

```diff
@@ -33,6 +33,13 @@ var promptIconsFunc = func(icons *survey.IconSet) {
 	icons.Question.Text = "▸"
 }
 
+// Env variables found in GCP, AWS and Azure cloudshell
+var (
+	gcpCloudEnv   = "CLOUD_SHELL"
+	awsCloudEnv   = "AWS_EXECUTION_ENV"
+	AzureCloudEnv = "POWERSHELL_DISTRIBUTION_CHANNEL"
```

Comment:
Is this available both in the Azure powershell and bash terminals?

Reply:
![tenor-44026482](https://user-images.githubusercontent.com/5712253/129904477-6206cf2a-1a8d-4bf4-8306-a6fe568782cc.gif)

Reply:
It does indeed. Confirmed this yesterday. Salim raised the same question.